src/interface/report/EventParser.js

```diff
@@ -127,7 +128,7 @@ class EventParser extends React.PureComponent {
         this.stopFakeNetworkProgress();
         timeAvailable && console.time('full parse');
         const parser = new parserClass(report, player, fight, combatants, characterProfile);
-        return this.parseEvents(parser, report, player, fight, events);
+        return this.parseEvents(parser, report, player, fight, [...bossEvents, ...events]);
```

Comment:
We're going to need this before the parse events method is called for phase selection in the future, would it be possible to make a list of all the phasestart/phaseend events before this and then pass that as-is to the events array? i.e. Instead of determining the events in the normalizer, how about doing it on the line above this and just letting the normalizer insert the events at the correct index?